dangerzone/gui/main_window.py

```diff
@@ -40,7 +40,7 @@
 
 # FIXME: Make the link clickable.
 UPDATE_SUCCESS_MSG_OUTRO = """\
-Please visit https://dangerzone.rocks to install this update.
```

Comment:
Maybe the link with the anchor? https://dangerzone.rocks/#downloads
Worst-case scenario, if the anchor is changed it just goes to the top of the page.

Reply:
Fixed in d6cb727.